sentry-sample/src/main/AndroidManifest.xml

```diff
@@ -34,7 +34,7 @@
     <meta-data android:name="io.sentry.anr.timeout-interval-mills" android:value="2000" />
 
     <!-- Raise ANR events even if the debugger is attached -->
-    <meta-data android:name="io.sentry.anr.report-debug" android:value="true" />
+    <meta-data android:name="io.sentry.anr.report-debug" android:value="false" />
```

Comment:
this is annoying if we are debugging and have breakpoints, it should be disabled by default.

Reply:
Can also just comment out the whole line (it's disabled by default) e, leave it with a comment (to cause ANR while debugging)

Reply:
you are right, I will comment then and keep as it was, so one will know how to enable it.